src/esm_runscripts/filedicts.py

```diff
@@ -115,10 +117,24 @@ def __init__(self, full_config: dict, attrs_address: dict):
             full_config, attrs_address, separator=".", default={}
         )
         super().__init__(attrs_dict)
+        self._original_filedict = copy.deepcopy(attrs_dict)
         self._config = full_config
         self.name = attrs_address.split(".")[-1]
         self.component = component = attrs_address.split(".")[0]
         self.path_in_computer = pathlib.Path(self["path_in_computer"])
+        self.all_model_filetypes = full_config["general"].get("all_model_filetypes", [
+            "bin",
+            "config",
+            "forcing",
+            "input",
+            "couple",
+            "log",
+            "outdata",
+            "restart",
+            "ignore",
+        ])
```

Comment:
If we are going to use this list again, maybe it would be a good idea to put them into another list like `default_filetypes = ["bin", "forcing", ...]` and use it instead. That would remove the duplications in the future.

Reply:
I don't get this, this should we always be provided by `config["general"]["all_model_filetypes"]`, the main reason for me to use a `get` here, was to avoid problems with the current unite tests.

Reply:
What I mean is that, whenever I see a long list passed to a function, I would give it a name. So that it is both more readable and reusable.